CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.16
+### Added
+- Enabled state transitions without modification of models.
+
+### Fixed
+=======
```

Comment:
please remove this line

Reply:
> please remove this line

Thanks for the review comment @tunacicek . 

I have removed the line.